ValetTests/ValetTests.m

```diff
@@ -110,8 +110,11 @@ - (void)test_initialization_invalidArgumentsCauseFailure;
 {
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"" accessibility:VALAccessibilityAlways]);
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"test" accessibility:0]);
-    XCTAssertNil([[VALSynchronizableValet alloc] initWithIdentifier:@"test" accessibility:VALAccessibilityWhenPasscodeSetThisDeviceOnly]);
```

Comment:
`VALAccessibilityWhenUnlockedThisDeviceOnly` is correct (or rather, incorrect, which is what we're testing) for both.

`VALSynchronizableValet` can not be `ThisDeviceOnly`, and `VALSecureEnclaveValet` must be `WhenPasscodeSetThisDeviceOnly`, so `VALAccessibilityWhenUnlockedThisDeviceOnly` is guaranteed to fail.